docs/tutorial/creating_your_first_admin_class/defining_entities.rst

```diff
@@ -256,6 +256,50 @@ Comment
         }
     }
 
+.. note::
+
+    For advanced usage, ``$id`` might be implemented as an object. The bundle will automatically resolve its string
+    representation from the ID object using ``$entity->getId()->__toString()`` (if implemented) when needed
+    (e.g., for generating url / rendering).
+
+    For example, in a use case where `InnoDB-optimised binary UUIDs`_ is implemented:
+
+    .. code-block:: php
+
+        <?php
+
+        class Comment
+        {
+            /**
+             * @var \Ramsey\Uuid\UuidInterface
+             * @Id
+             * @Column(type="uuid_binary_ordered_time", unique=true)
+             * @GeneratedValue(strategy="CUSTOM")
+             * @CustomIdGenerator(class="Ramsey\Uuid\Doctrine\UuidOrderedTimeGenerator")
+             */
+            protected $id;
+
+            // ...
+        }
+
+
+    .. code-block:: php
+
+        <?php
+
+        // Retrieved from \Ramsey\Uuid\Uuid
+        class Uuid implements UuidInterface
```

Comment:
what don't you add `use ....` instead of comment?

Reply:
Like using Trait? Not sure if I understand.

Looks like my documentation here is a bit confusing. Any suggestion to approach this? it is a copy from `\Ramsey\Uuid\Uuid` to show that it is the implementation of `\Ramsey\Uuid\UuidInterface` and has offered the __toString method.

Reply:
I have given it another attempt now. This might clarify a bit?

Reply:
I meant as import the interface instead of commenting about it